packages/jest/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@vanilla-extract/jest",
```

Comment:
Did you consider calling this `jest-transform`?

Reply:
Both you and Mike raised this haha. I was on the fence, think I'll update it.